builder/dockerfile/evaluator_unix.go

```diff
@@ -4,6 +4,6 @@ package dockerfile
 
 // platformSupports is a short-term function to give users a quality error
 // message if a Dockerfile uses a command not supported on the platform.
-func platformSupports(command string) error {
+func platformSupports(command interface{}) error {
```

Comment:
I'm not sure we want to let implementation decide what parts of the `Dockerfile` they implement. All implementations should have the same behavior for user commands.